hack/update-codegen.sh

```diff
@@ -51,12 +55,19 @@ function print_usage() {
     echo "      Verifies that the generated code is up-to-date. The existing generated"
     echo "      code will not be touched."
     echo ""
+    echo "  -m, --only-mocks"
```

Comment:
The "only XXX" approach makes it impossible to select more than one type.

I propose to call this option `--mocks`. In the future we could add further options to select other types, e.g. `--tekton-client`. You can combine those options to generate more than one type but not all. If none of those options is specified, all types get generated.